torchinductor/scheduler.py

```diff
@@ -458,6 +561,12 @@ def __init__(self, nodes):
                 assert False, node
         self.name_to_node = {node.get_name(): node for node in self.nodes}
 
+        if INDUCTOR_SCHEDULER_GRAPH:
+            from functorch._src.aot_autograd import get_graph_being_compiled
+
+            graph_name = get_graph_being_compiled()
+            create_fx_graph(self.nodes, graph_name, print_graph=True)
```

Comment:
not sure you want `print_graph=True` here?

Reply:
as @Chillee said below, this line is only called when we are in debugging mode, so I think printing out the graph is probably helpful?